ci/scripts/clone-build_ci.sh

```diff
@@ -73,37 +68,53 @@ if [[ "${pr_state}" != "OPEN" ]]; then
 fi  
  
 # checkout pull request
-"${GH}" pr checkout "${PR}" --repo "${repo_url}"
+"${GH}" pr checkout "${PR}" --repo "${REPO_URL}"
 
 # get commit hash
 commit=$(git log --pretty=format:'%h' -n 1)
 echo "${commit}" > "../commit"
 
 # run build script
-cd sorc
+cd sorc || exit 1
 export BUILD_JOBS=8
 rm -rf log.build
```

Comment:
Can you please move these 2 lines right before the call to `./build_all.sh`?  It makes sense there.